pkg/client/client.go

```diff
@@ -124,7 +137,14 @@ func (c *Client) GetIPFSAddress() string {
 	// by default use .env if it exists
 	_, err := os.Stat(env_path)
 	if errors.Is(err, os.ErrNotExist) {
-		return IPFS_ADDRESS
+		switch c.clientMode {
+		case mt.ClientMode_ENDPOINT_BETA:
+			return IPFS_ADDRESS_BETA
+		case mt.ClientMode_ENDPOINT_DEV:
+			return IPFS_ADDRESS_DEV
+		case mt.ClientMode_ENDPOINT_LOCAL:
```

Comment:
Add prod as an option for client mode here

Reply:
done